docs/userguide.md

```diff
@@ -294,6 +294,9 @@ Currently you could adjust these configs in your spark.conf file.
 | --------   | -----:   | :----: |
 | spark.tispark.statistics.auto_load | true | Whether to load statistics info automatically during database mapping. |
 
+## Reading partition table from in TiDB
+Currently, only range partition table is limited supported. 
```

Comment:
which `partition function` do we support?

Reply:
what if read data from partition table but the function is not supported? throw error?

Reply:
updated. I forget to push the newly commit.